draft-ietf-cbor-7049bis.md

```diff
@@ -340,8 +340,7 @@ of a number of simple values and tags right in this document, such as:
 * `false`, `true`, `null`, and `undefined` (simple values identified by 20..23)
 * integer and floating point values with a larger range and precision
   than the above (tags 2 to 5)
-* application data types such as a point in time or an RFC 3339
-  date/time string (tags 1, 0)
+* application data types such as a point in time (tags 1, 0)
```

Comment:
Not sure an RFC 3339 date/time string always is a "point in time"

Reply:
Tag 0 refers to Section 3.3 of [RFC4287], which uses the date-time production from RFC3339, which has all of an hour, minute, and second. I'm not sure what else that is besides a point in time?

Reply:
I'm also happy to remove this particular change if that unblocks the rest of the patch.

Reply:
Hi Jeffrey — this is not stuck on this observation (which I think I already agreed with in email), but my editing time is currently being absorbed by an AUTH48.  Once this is done, we'll get to it...  Sorry for the latency.